### ISSUE TITLE
[Docummentation] Is there a way to generate a book for it?

### ISSUE THREAD

**Issue by weyderfs** [concourse/docs#512]
Hi guys,

Is there some e-book created for Concourse CI? Actually the docs just are in `HTML` [page](https://concourse-ci.org/docs.html). But I would like export a `.pdf` or `.epub` to read in my Kindle.

I've searched in the framework project of [Booklit](https://booklit.page/), but not found nothing about that. 😢

**Comment by taylorsilva** [comment]
There isn't and I don't think anyone has ever thought of doing that before! You could generate all the pages locally and then convert the html files to pdf maybe?

**Comment by weyderfs** [comment]
> There isn't and I don't think anyone has ever thought of doing that before! You could generate all the pages locally and then convert the html files to pdf maybe?

@taylorsilva I've imagined, but the framework in use to generate the _docs_ have this feature? 

Another point is, to generate locally the page locally I just need clone this repository and run the framework or there's a more easy way?

**Comment by taylorsilva** [comment]
Yup, run the scripts/build script and it'll generate all the html files
